lib/pipeline.runner.js

```diff
@@ -131,6 +132,12 @@ class PipelineRunner {
             pipeline_helper_1.PipelineHelper.EnsureValidPipeline(projectName, pipelineName, releaseDefinitions);
             let releaseDefinition = releaseDefinitions[0];
             logger_1.Logger.LogPipelineObject(releaseDefinition);
+            // Create ConfigurationVariableValue objects from the input variables
+            let variables = JSON.parse(this.taskParameters.azurePipelineVariables);

```

Comment:
Looks like the js needs rebuilt here to include your latest change regarding safer use of the JSON.parse

Reply:
yeah